app/services/userService.js

```diff
@@ -22,6 +24,25 @@ exports.findUserByEmail = email => UserModel.findOne({ where: { email: formatEma
 
 exports.findUserById = id => UserModel.findByPk(id);
 
+exports.findTopWeetAuthor = async () => {
+  try {
+    const mostWordWeetAuthor = await WeetModel.findAll({
+      where: {},
+      order: [[sequelize.fn('length', sequelize.col('content')), 'DESC']]
+    });
```

Comment:
This query would be giant in a production environment. You should add a limit 1 constraint for this kind o queries.